mango-core/src/main/java/org/calrissian/mango/net/MoreInetAddresses.java

```diff
@@ -321,6 +323,41 @@ public static Inet6Address decrement(Inet6Address ip) {
         return getInet6Address(decrementBytes(ip));
     }
 
+    /**
+     * Returns {@code true} if {@code address} is within the bounds of the {@code cidr}.
+     *
+     * @param address   the {@link InetAddress} to check
+     * @param cidr      the {@link CidrInfo} block to check bounds against
+     * @return          {@code true} if {@code address} is within the {@code cidr} range. {@code false} otherwise.
+     */
+    public static Boolean isInCidrRange(InetAddress address, CidrInfo cidr) {
```

Comment:
Why not just add a contains method to the CidrInfo object instead of making more methods for MoreInetAddresses?